ZenPacks/zenoss/LinuxMonitor/LinuxService.py

```diff
@@ -0,0 +1,25 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+
+"""LinuxService module.
+
+All custom behavior for Linux OS Service is defined here.
+
+"""
+
+
+from . import schema
+
+
+class LinuxService(schema.LinuxService):
```

Comment:
Normally we start at the Relationship level to see if we need to inherit.
If there is a question about it, fire up a conversation with the lead on this.
If its not broken as is, don't fix it.